### PR TITLE
Copyedit interactive data visualization dashboard

### PR DESCRIPTION
**Pull request by charlottejmc**
I've now prepared my copyedits for en/interactive-data-visualization-dashboard, represented in Issue #609.

### CODE REVIEW

File: en/drafts/originals/interactive-data-visualization-dashboard.md [hunk 1]
```diff
@@ -22,110 +22,115 @@ doi: XX.XXXXX/phen0000
...
+### Case Study
 
-Quantitative methods for content analysis (CA) have long been a tradition in mass communication studies, and the method of algorithmic text analysis (ATA) has become popular given the rising availability of large amounts of textual data.[^2] Both methods aim to infer meanings from text through classification or measurement. Whereas CA relies heavily on a carefully crafted codebook based on research questions and multiple human coders to ensure the reliability and validity of a systematic analysis,[^3] [^4] ATA relies on algorithms and models (a more general term for this method is [text mining](https://en.wikipedia.org/wiki/Text_mining) or [natural language processing](https://en.wikipedia.org/wiki/Natural_language_processing)).[^5] 
+The case study investigates how U.S. television stations have covered the current [war between Russia and Ukraine](https://en.wikipedia.org/wiki/Russo-Ukrainian_War). For example, we could compare the frequency of Ukraine-related keywords to Russia-related keywords employed by different stations. We could also compare the amount of coverage of the war between certain stations.
```
Comment (charlottejmc) [inline en/drafts/originals/interactive-data-visualization-dashboard.md:53]: I've added the wikipedia link https://en.wikipedia.org/wiki/Russo-Ukrainian_War to give context to our readers, but would you prefer the following article instead? https://en.wikipedia.org/wiki/Russian_invasion_of_Ukraine

File: en/drafts/originals/interactive-data-visualization-dashboard.md [hunk 2]
```diff
@@ -22,110 +22,115 @@ doi: XX.XXXXX/phen0000
...
+Mass communication studies have traditionally relied on quantitative Content Analysis (CA) methods. However, algorithmic text analysis (ATA) has also recently grown in popularity, due to the rising availability of large amounts of textual data.[^2] Both types of methods aim to infer meanings from text through classification or measurement. Whereas CA relies heavily on a carefully crafted codebook, built around research questions and verified by multiple human coders,[^3] [^4] ATA relies on computational methods like statistics and machine learning. You might have heard for example of [text mining](https://en.wikipedia.org/wiki/Text_mining), or [Natural Language Processing](https://en.wikipedia.org/wiki/Natural_language_processing).[^5]
```
Comment (charlottejmc) [inline en/drafts/originals/interactive-data-visualization-dashboard.md]: I reworded this paragraph quite a bit, so I wanted to check with you that it still expresses your original meaning. I was a little confused by the first sentence especially. I also didn't think 'text mining' or 'natural language processing' could really be called 'more general terms' for ATA – the latter sounds like a much more general term, of which the former two would be specific instantiations! What do you think?

File: en/drafts/originals/interactive-data-visualization-dashboard.md [hunk 3]
```diff
@@ -422,137 +447,143 @@ line_fig_rus.update_layout(
...
+## Further Practice: Second Example
 
-Second, scroll down and find the section called "Environment Variables." Click "Add Environment Variable" and input `PYTHON_VERSION` as the key and the Python version that you use on your machine as the value (use `$python -V` in command line to check your Python version). If you don't explicitly specify the Python version this way, Render will use Python 3.7 as default, which may cause conflicts between this old Python version and the libraries' versions specified in `requirements.txt`.
+To demonstrate the wide applicability of the approach used in the case study above, this lesson will show you how to create another dashboard, using a different dataset. This second example explores a different research question: 'What were the top 10 languages used by non-English U.S. newspapers by decade from the 1690s to today?' We'll design a dashboard to show the top ten languages in each decade dating back to the 1690s, highlighting any shifts in their rankings, and the emergence or decline of different languages over time.
```
Comment (charlottejmc) [inline en/drafts/originals/interactive-data-visualization-dashboard.md:514]: I tried changing the wording of the second research question here, but I still think it is very tricky to try to express your particular question in a clear way. Do you prefer the original? Maybe we can brainstorm some options...

**Comment by charlottejmc** [comment]
Hello @hluling and @caiocmello,

 I've now prepared the copyedits for this lesson. I'd be grateful if you could review the adjustments and confirm that you are happy for me to merge these. You can see the details of my edits under the [files changed](https://github.com/programminghistorian/ph-submissions/pull/642/files) tab!

I'd like to bring your attention to the comments I have attached to specific lines. You can respond to any of my suggestions via the comments below, or click _Resolve conversation_ if you're happy with them like this:

image

If you want to make edits, please work [here](https://github.com/programminghistorian/ph-submissions/pull/642/files) by clicking the three dots at the upper right of the file, then _Edit file_:

image

**Comment by charlottejmc** [comment]
One other thing, @hluling: your lesson will need a very short abstract (1-2 sentences max) to give a rapid overview of what it teaches. Could you prepare this and either send it to me via a comment, or [add it directly in this branch](https://github.com/programminghistorian/ph-submissions/pull/642/files) to the `abstract` field in the markdown file's YAML header? Thank you!

File: en/drafts/originals/interactive-data-visualization-dashboard.md (hunk 1 above)
Comment (hluling) [reply en/drafts/originals/interactive-data-visualization-dashboard.md:53]: Thanks! The current link works better.

File: en/drafts/originals/interactive-data-visualization-dashboard.md (hunk 2 above)
Comment (hluling) [reply en/drafts/originals/interactive-data-visualization-dashboard.md]: Thanks. Please see [my edit here](https://github.com/programminghistorian/ph-submissions/pull/642/commits/5c49fe96a3cff154375582e00dbff058fa9f4bd0). Is the current version clear enough?

File: en/drafts/originals/interactive-data-visualization-dashboard.md (hunk 3 above)
Comment (hluling) [reply en/drafts/originals/interactive-data-visualization-dashboard.md:514]: I think your version is better!

**Comment by hluling** [comment]
Hi @charlottejmc, thanks for the edits! There is just one unresolved comment, please see above. Everything else looks good to me!

Abstract: This lesson demonstrates how to create interactive web-based dashboards using Python's Dash library, focusing on two case studies in media studies. By walking through the process of data retrieval, visualization, and web deployment, the lesson provides a practical guide for digital humanities researchers to make their research outputs more accessible and engaging through interactive data visualization.

File: en/drafts/originals/interactive-data-visualization-dashboard.md (hunk 2 above)
Comment (charlottejmc) [reply en/drafts/originals/interactive-data-visualization-dashboard.md]: Thank you @hluling, I've incorporated some of your changes into [this latest version](https://github.com/programminghistorian/ph-submissions/pull/642/commits/f618a9c39d51b20701d5ad49a4e4ded529f528bc).

**Comment by charlottejmc** [comment]
Thank you very much, @hluling! I'll now merge this copyedit branch so that your lesson can move to the next step in our publishing process, which is typesetting.

**Pull request merged**